lib/auth/webauthncli/fido2.go

```diff
@@ -77,7 +77,7 @@ var fidoNewDevice = func(path string) (FIDODevice, error) {
 
 // IsFIDO2Available returns true if libfido2 is available in the current build.
 func IsFIDO2Available() bool {
-	val, ok := os.LookupEnv("TSH_FIDO2")
+	val, ok := os.LookupEnv("TELEPORT_FIDO2")
 	// Default to enabled, otherwise obey the env variable.
 	return !ok || val == "1"
```

Comment:
If you used `strconv.ParseBool` here instead of checking for a literal `"1"` then you could use `Getenv` instead of `LookupEnv` and you would accept more inputs.

Reply:
TBH I kind of prefer the reduced set of values. It's easy to expand the set of values, but hard to reduce once we commit to something.